src/iologindata.cpp

```diff
@@ -763,24 +763,22 @@ bool IOLoginData::savePlayer(Player* player)
 		return false;
 	}
 
-	if (player->lastDepotId != -1) {
```

Comment:
:roll_eyes:

Reply:
yep, this code is as useless as your comments

Reply:
You're a funny little guy. Have fun!

Reply:
For other people though (cause seems like you didn't learn how to communicate properly yet)

Yeah idea is good, implementation not exactly, we could store "save bool" for every single depot and then decide if they need a save or not.

I did something similar here: https://github.com/nekiro/TFS-1.4-Downgrades/commit/e8614e2157a7120d7fb6fd6786e675ac6031097f

Reply:
Yes, so because you don't like the implementation, you decide to remove it completely. It is, indeed, a great enhancement. Let's just re-save every player's depot everytime.

Reply:
It was a fast implementation, just to have working "new" depots, you think this will be merged right away or what? It awaits reviews and will be worked on

Reply:
Yes, That's what I thought and that's what has happend :rose: